docs/example-input-event.md

```diff
@@ -11,30 +11,25 @@ sidebar_label: Input Event
 > [`user-event`](ecosystem-user-event.md)
 
 ```jsx
-import React, {useState} from 'react'
+import * as React from 'react'
+import { useState } from 'react'
```

Comment:
What is the point here? React.useState seems sufficient since `react` isn't tree-shakeable anyway. Note that react isn't published as an ES6 module and anything but `const React = require('react')` is just makeup.

Reply:
This is a first step into removing React import statements, which won't be necessary anymore with React 17

Reply:
Ok. Why do we need to import twice?

Reply:
I've done this, so we can easily remove all `import * as React from 'react'` lines whenever we update everything to React 17

Reply:
But it's confusing now for readers so that we have to do less work now. Seems like we're optimizing at the wrong end. Especially since this is markdown only where it doesn't matter. 

`import * as React from 'react'` works in both 16 and 17 so I'm trying to understand who is benefiting from the style you're proposing.

Reply:
I'll revert all double imports then 🙂